terraform/ecs.tf

```diff
@@ -12,7 +13,7 @@ data "aws_ami" "ecs" {
 
   filter {
     name   = "architecture"
-    values = ["x86_64"]
+    values = ["${each.value}"]
```

Comment:
can be    
values = [each.value] 

(otherwise terraform will show a deprecation warning)